390/Serena_Li.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: "MP2 Viz"
+author: "Serena Li"
+date: "9/15/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(dplyr)
+library(readr)
+
+```
+
+```{r}
+stem <- read_csv("tree_main_census/data/census-csv-files/scbi.stem1.csv")
+```
+
+#barplot
+```{r}
```

Comment:
```suggestion
```{r, fig.height = 10}
```
Here I'm making an explicit suggestion.

Reply:
Click on Commit Suggestion to commit it, then do a local pull of the repo and see the effect. It will make the plot taller than default, thus making the plot a little less cramped.